### PR TITLE
LF Parser: handle interface

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by cocreature — APPROVED** [review]
Nice, thank youk

File: daml-lf/parser/src/main/scala/com/digitalasset/daml/lf/testing/parser/ExprParser.scala [hunk 1]
```diff
@@ -324,7 +325,12 @@ private[parser] class ExprParser[P](parserParameters: ParserParameters[P]) {
     "BIGNUMERIC_TO_TEXT" -> BBigNumericToText,
   )
 
-  private lazy val experimental: Parser[Expr] =
+  private lazy val eCallInterface: Parser[ECallInterface] =
+    Id("icall") ~! `@` ~> fullIdentifier ~ id ~ expr0 ^^ { case ifaceId ~ name ~ body =>
```
Comment (sofiafaro-da) [inline daml-lf/parser/src/main/scala/com/digitalasset/daml/lf/testing/parser/ExprParser.scala:329]: Why `icall` and not `call_interface`? Seems a bit odd to abbreviate this one.

Comment (remyhaemmerle-da) [reply daml-lf/parser/src/main/scala/com/digitalasset/daml/lf/testing/parser/ExprParser.scala:329]: I will sneak  that in another PR.
What do you think about `call_method` ?

Comment (sofiafaro-da) [reply daml-lf/parser/src/main/scala/com/digitalasset/daml/lf/testing/parser/ExprParser.scala:329]: That works, but personally I'd like it to match the AST. We can change the AST to  `ECallMethod` if you like?

Comment (remyhaemmerle-da) [reply daml-lf/parser/src/main/scala/com/digitalasset/daml/lf/testing/parser/ExprParser.scala:329]: We could have a whole pass on naming before release.

**Pull request merged**